compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/Clickable.kt

```diff
@@ -840,13 +833,13 @@ private class ClickableSemanticsNode(
     }
 }
 
-private sealed class AbstractClickablePointerInputNode(
+internal sealed class AbstractClickablePointerInputNode(
     protected var enabled: Boolean,
     protected var interactionSource: MutableInteractionSource?,
     protected var onClick: () -> Unit,
     protected val interactionData: AbstractClickableNode.InteractionData
 ) : DelegatingNode(), ModifierLocalModifierNode, CompositionLocalConsumerModifierNode,
-    PointerInputModifierNode {
+    PointerInputModifierNode, FocusRequesterModifierNode {
 
```

Comment:
Does it also implicitly affect Modifier.toggle?
So elements with toggle would also be focused when clicked?

Reply:
Yes, `toggleable` use `clickable`. I checked, it works for them

Reply:
Thanks!